examples/notebooks/simulating-long-experiments.ipynb

```diff
@@ -0,0 +1,2011 @@
+      "text/plain": [
+       "<Figure size 720x720 with 9 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "vars_to_plot = [\n",
+    "    \"Capacity [A.h]\",\n",
+    "    \"Loss of lithium inventory [%]\",\n",
+    "    \"Loss of capacity to negative electrode SEI [A.h]\",\n",
+    "    \"Loss of active material in negative electrode [%]\",\n",
+    "    \"Loss of active material in positive electrode [%]\",\n",
+    "    \"x_100\",\n",
+    "    \"x_0\",\n",
+    "    \"y_100\",\n",
+    "    \"y_0\"\n",
+    "]\n",
+    "l = len(vars_to_plot)\n",
+    "n = int(l//np.sqrt(l))\n",
+    "m = int(np.ceil(l/n))\n",
+    "\n",
+    "fig, axes = plt.subplots(n,m,figsize=(10,10))\n",
```

Comment:
very nice notebook! I think this will be popular. It doesn't need to be for this issue, but might be good to automate this e.g. `sim.plot_summary_variables()`

Reply:
Yeah, though it's not too hard to plot the variables (they're just numpy arrays, unlike the outputs of a regular solution) so I like leaving customizability to the user